Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -93,6 +93,25 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         let cardRequest = CardRequest(orderID: orderID, card: card, sca: .scaAlways)
         cardClient.approveOrder(request: cardRequest)
     }
+    
+    func vaultCard(
+        card: Card,
+        customerID: String? = nil
+    ) async {
```

Comment:
I don't think this will hit our linter line length so we should be able to single line it:
```suggestion
    func vaultCard(card: Card, customerID: String? = nil) async {
```